tree/dataframe/inc/ROOT/RDF/RDatasetSpec.hxx

```diff
@@ -29,11 +29,23 @@ class RLoopManager;
 namespace RDF {
 namespace Experimental {
 
+// clang-format off
 /**
+\class ROOT::RDF::Experimental::RDatasetSpec
 \ingroup dataframe
-\brief A dataset specification for RDataFrame.
+\brief The dataset specification for RDataFrame.
+
+This class allows users to create the dataset specification for RDataFrame 
+to which they add samples (using the RSample class object) with tree names and file names, 
+and, optionally, the metadata information (using the RMetaData class objects). 
+Adding global friend trees and/or setting the range of events to be processed
+are also available.
+
+Note, there exists yet another method to build RDataFrame from the dataset information using the JSON file format: ROOT::RDataFrame FromSpec(const std::string &jsonFile). 
```

Comment:
Can we put a link to `FromSpec`, I think using the full doxygen URL should be ok 

https://root.cern/doc/master/namespaceROOT_1_1RDF_1_1Experimental.html#a7193987f3c1b65c649399656cc6acce8

Reply:
I built the documentation html locally and the link was already working by just putting the function name like it is now

Reply:
Alright, perfect then!